compiler/x/codegen/FPTreeEvaluator.cpp

```diff
@@ -1265,27 +1240,25 @@ TR::Register *OMR::X86::TreeEvaluator::fpConvertToLong(TR::Node *node, TR::Symbo
 
       // For slow conversion only, change the rounding mode on the FPU via its control word register.
       //
-      if (!optLevel)
-         {
-         int16_t fpcw = comp->getJittedMethodSymbol()->usesSinglePrecisionMode() ?
-                           SINGLE_PRECISION_ROUND_TO_ZERO : DOUBLE_PRECISION_ROUND_TO_ZERO;
+      {
```

Comment:
Is there any need for a new scope here? Can it be removed?

Reply:
No reason--just lazy in the refactoring.  I'll fix.  Thanks for keeping me honest!